packages/foreman/foreman/foreman.req

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/python2
+
+"""
+Automatic requires generator for webpack bundles
+"""
+
+import json
+import os
+import sys
+
+FOREMAN_MANIFEST = '/var/lib/foreman/public/webpack/manifest.json'
+
+paths = [path.rstrip() for path in sys.stdin.readlines()]
+
+def foreman_webpack(name, chunk):
+    filename, fileext = os.path.splitext(chunk)
+    fileext = fileext.lstrip('.')
+    return 'foreman-webpack-{}({}) = {}'.format(fileext, name, filename)
+
+for path in paths:
+    if path.endswith('/manifest.json'):
```

Comment:
This is a check to ensure that we only generate the requires if the plugin uses webpack, correct?

Reply:
yes and no, it's a safenet. non-webpack plugins should not even run this code in the first place, because they don't have a path that would trigger it (see `foreman.attr`).